components/esp_wifi_remote/wifi_remote_net.c

```diff
@@ -0,0 +1,90 @@
+/*
+ * SPDX-FileCopyrightText: 2023-2024 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <esp_private/wifi.h>
```

Comment:
private may create issues?

Reply:
We need to keep it as is, since we're using wifi netifs as esp_hosted channels. we need to use `wifi_internal_` APIs for it.